go.mod

```diff
@@ -168,6 +168,7 @@ require (
 	github.com/ugorji/go/codec v1.2.6 // indirect
 	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c // indirect
 	go.uber.org/atomic v1.10.0 // indirect
+	golang.org/x/exp v0.0.0-20221012211006-4de253d81b95
```

Comment:
Using this library in pre-loads to easily get map keys. It seems generally useful when working with maps.